phing/tasks/setup.xml

```diff
@@ -206,6 +206,16 @@
   </target>
 
   <target name="setup:update" description="Update current database to reflect the state of the Drupal file system.">
+    <!-- Enable features module. -->
+    <drush command="en" assume="yes" alias="${drush.alias}">
+      <param>features</param>
```

Comment:
Additionally, I think we should run this target in BLT's .travis.yml after `blt acsf:init` to at least verify that it does not return a non-zero exit code.